index.html

```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rizwan Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+        body{background: url("Image/backimg.png");
+        background-repeat: no-repeat;
+        background-attachment: fixed;
+        background-size: cover;
+    }     
+    </style>
+</head>
+<body>
+
+    <header> 
+        <nav>
+            <div class="nav-menu-content">
+
+                <div class="logo"><a class=mylogo> My Logo </a></div>
+    
+                <div class="nav-menu-content">
+                    <ul class="nav-menu">
+                        <li class="nav-items"><a href="#">Portfolio</a></li>
+                        <li class="nav-items"><a href="#">About</a></li>
+                        <li class="nav-items"><a href="#">Contact</a></li>
+                      </ul>
+                      
+                
+                        <div class="hamburger">
+                            
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                        </div>
+                            
+                </div>
+            </div>
+        </nav>
```

Comment:
- [ ] Nice job adding the `nav` tag. But according to a suggestion from the previous reviewer, you should change (replace) the `<div class="nav-menu-content">` with `nav` tag. I think you can do it easily and then just add your class name `nav-menu-content` to the `nav` tag to avoid missing styling.

Reply:
nav-menu -content replaced with <nav>